routes/cube_routes.js

```diff
@@ -850,12 +853,14 @@ router.get('/playtest/:id', async (req, res) => {
 
 router.get('/analysis/:id', async (req, res) => {
   try {
-    const cube = await Cube.findOne(buildIdQuery(req.params.id)).lean();
+    const fields = 'cards name owner defaultDraftFormat draft_formats card_count type';
+    const cube = await Cube.findOne(buildIdQuery(req.params.id), fields).lean();
```

Comment:
why create a variable here, would be cleaner to just pass the string in directly